### PR TITLE
10.0 pos partner deselection publication

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Comment by yelizariev** [comment]
@KolushovAlexandr second commit has wrong link to PR. Either fix it (``it-projects-llc/addons-dev#647``, it-projects-llc/addons-dev#647) or remove, please

File: pos_partner_deselection/models/models.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Copyright 2019 Kolushov Alexandr <https://it-projects.info/team/KolushovAlexandr>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models, fields
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    customer_deselection_interval = fields.Integer(string='Customer Deselection Interval', default=0)
```
Comment (yelizariev) [inline pos_partner_deselection/models/models.py]: It's not clear from UI is it seconds or minutes

File: pos_partner_deselection/static/src/js/test_pos_deselection.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        }, {
+            trigger: ".o_pos_kanban button.oe_kanban_action_button",
+            content: _t("<p>Click to start the point of sale interface. It <b>runs on tablets</b>, laptops, or industrial hardware.</p><p>Once the session launched, the system continues to run without an internet connection.</p>"),
+            position: "bottom"
+        }, {
+            content: "Switch to table or make dummy action",
+            trigger: '.table:not(.oe_invisible .neworder-button), .order-button.selected',
+            position: "bottom",
+            timeout: 30000,
+        }];
```
Comment (yelizariev) [inline pos_partner_deselection/static/src/js/test_pos_deselection.js:24]: Why don't you add customer selection and checking deselection?

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_partner_deselection/static/src/js/test_pos_deselection.js (hunk 2 above)
Comment (KolushovAlexandr) [reply pos_partner_deselection/static/src/js/test_pos_deselection.js:24]: Because it might break other tours due to deselected partner before a payment

Comment (yelizariev) [reply pos_partner_deselection/static/src/js/test_pos_deselection.js:24]: No, it cannot

**Review by yelizariev — APPROVED** [review]

**Pull request merged**